web/l10n/en.json

```diff
@@ -90,6 +90,7 @@
     "attributeDevicePassword": "Device Password",
     "attributeDeviceInactivityStart": "Device Inactivity Start",
     "attributeDeviceInactivityPeriod": "Device Inactivity Period",
+    "attributeDeviceAllowForward": "Allow Forward",
```

Comment:
This should probably be "Allow Forwarding" or something like this.

Reply:
Renamed to "Allow Forwarding"